mosviz/loaders/test/sample_test.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+def test_sample():
+    assert(1==1)
```

Comment:
What is the purpose of this test? If this is just a placeholder, you can also just do `assert True` or `pass`.